pkg/lib/dialogs.jsx

```diff
@@ -107,7 +107,8 @@ export const WithDialogs = ({ children }) => {
 
     const Dialogs = {
         show: setDialog,
-        close: () => setDialog(null)
+        close: () => setDialog(null),
+        isActive: () => dialog !== null
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19230-20230824-094945-41a2db55-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)